### PR TITLE
[Feat] image and input components added

### PR DESCRIPTION
**Pull request by t007rushi**
This is PR for 
Image component : 
One square and one round image both responsive

input: 
There is textarea input fiels
One valid and one error styling inpt have added

### CODE REVIEW

File: components/image.css [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+}
+
+.img-resp {
+  width: 100%;
+  height: auto;
+  object-fit: cover;
+}
+
+.img-round {
+  border-radius: 100%;
```
Comment (vijaytembugade) [inline components/image.css:16]: border-radius : 50% or border-radius :100% ??? 
Can you comfirm it?

**Review by vijaytembugade — APPROVED** [review]
other things Looks perfect.

**Comment by jatinmore** [comment]
-Iframe div getting outside of the main content 
-Add some Padding to main content
-All other things are good

**Review by Raghav888 — APPROVED** [review]
Approved

**Review by MarkVed17 — APPROVED** [review]
Good to proceed!

File: components/image.css [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+width:40%;
+flex-wrap: wrap;
+gap: 2rem;
+margin: auto;
+}
+
+.img-resp {
+  width: 100%;
+  height: auto;
+  object-fit: cover;
```
Comment (Priya-Patel31) [inline components/image.css]: use object-fit : contain rather than object-fit : cover !!

File: components/input.css [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+  flex-direction: column;
+  flex-wrap: wrap;
+  justify-content: center;
+  align-items: center;
+  gap: 1rem;
+  padding: 1rem;
+}
+
+.textbox {
+  display: flex;
```
Comment (Priya-Patel31) [inline components/input.css]: Just a suggestion that you could have made utility classes for this as this is most commonly used property .

**Review by Priya-Patel31 — CHANGES_REQUESTED** [review]
Make changes in image.css as it is necessary !!

File: components/image.css (hunk 1 above)
Comment (t007rushi) [reply components/image.css:16]: above 50% it becomes round

File: components/input.css (hunk 3 above)
Comment (t007rushi) [reply components/input.css]: done with changes plz approve

**Review by Priya-Patel31 — APPROVED** [review]
Changes have been done !! You can now merge it .

**Pull request merged**